app-api/src/main/java/org/openmrs/eip/app/logger/BinlogReaderOffsetErrorsAppender.java

```diff
@@ -0,0 +1,13 @@
+package org.openmrs.eip.app.logger;
+
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.AppenderBase;
+
+public class BinlogReaderOffsetErrorsAppender extends AppenderBase<ILoggingEvent> {
```

Comment:
Why is this necessary? Please document some of the new classes

Reply:
Please see [this comment](https://github.com/FriendsInGlobalHealth/openmrs-eip/pull/21#discussion_r1023580970) and share your opinion.